x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/components/table/column_header.tsx

```diff
@@ -14,28 +14,35 @@ interface Props {
   label: string;
   toolTip?: string;
   formula?: string;
+  showDocumentationLink?: boolean;
 }
 
-export const ColumnHeader = React.memo(({ label, toolTip, formula }: Props) => {
-  return (
-    <EuiFlexGroup gutterSize="xs">
-      <div
-        css={css`
-          overflow-wrap: break-word !important;
-          word-break: break-word;
-          min-width: 0;
-          text-overflow: ellipsis;
-          overflow: hidden;
-        `}
-      >
-        {label}
-      </div>
+export const ColumnHeader = React.memo(
```

Comment:
```typescript
export const ColumnHeader = React.memo(
  ({ label, toolTip, formula, showDocumentationLink = true }: Props) => (
    <EuiFlexGroup gutterSize="xs">
      <div
        css={css`
          overflow-wrap: break-word !important;
          word-break: break-word;
          min-width: 0;
          text-overflow: ellipsis;
          overflow: hidden;
        `}
      >
        {label}
      </div>

      {toolTip && (
        <Popover>
          <TooltipContent
            formula={formula}
            description={toolTip}
            showDocumentationLink={showDocumentationLink}
          />
        </Popover>
      )}
    </EuiFlexGroup>
  )
);
```

Total preference here, but this one can skip the `return` syntax.

Reply:
Nice, thanks for the suggestion ( I just added a prop here, it was using the `return` syntax before my change so I changed it as proposed 👍 )